cylc/flow/tui/data.py

```diff
@@ -20,7 +20,7 @@
           id
           name
         }
-        jobs {
+        jobs(sort: { keys: ["submit_num"], reverse: true}) {
```

Comment:
i.e. same as doing `.sort(lambda..., reverse=True)` but on the server side

Reply:
Not sure whether it's better to put load on the scheduler or on Tui.

Perhaps the data store could keep jobs in reverse order for efficiency.